package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "dsrm-frontend",
+  "private": true,
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "build": "vite build",
+    "preview": "vite preview",
+    "start": "vite",
+    "lint": "eslint . --ext .ts --ext .tsx",
+    "lint:fix": "eslint . --ext .ts --ext .tsx --fix",
+    "lint:watch": "esw . --watch --ext .ts --ext .tsx ",
+    "prettyprint": "prettier --check .",
+    "prettyprint:fix": "prettier --write ."
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/material": "^5.10.15",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "sass": "^1.56.1"
+  },
+  "devDependencies": {
+    "@eslint/create-config": "^0.4.1",
+    "@swc/cli": "^0.1.57",
```

Comment:
Let's remove swc dependencies as we are not using it now

Reply:
Removed in 2fe809c